packages/app/src/systems/Transaction/__mocks__/tx.ts

```diff
@@ -226,6 +230,61 @@ export const MOCK_TRANSACTION_CONTRACT_CALL: MockTransaction = {
   ],
 };
 
+const tenSecondsAgo = new Date(new Date().getTime() - 10 * 1000);
+export const MOCK_TRANSACTION_CONTRACT_CALL_SECONDS_AGO: MockTransaction = {
+  transaction: MOCK_TRANSACTION_CONTRACT_CALL.transaction,
+  tx: {
+    ...MOCK_TRANSACTION_CONTRACT_CALL.tx,
+    operations: [
+      {
+        ...MOCK_TRANSACTION_CONTRACT_CALL.tx.operations[0],
+        name: OperationName.mint,
+      },
+    ],
+    time: tenSecondsAgo.toString(),
+    id: '0x18617ccc580478214175c4daba11903df93a66a94aada773e80411ed06b6ade8',
+    status: TxStatus.pending,
+    type: TxType.script,
+  },
+  receipts: MOCK_TRANSACTION_CONTRACT_CALL.receipts,
+};
+
+const fourMinutesAgo = new Date(new Date().getTime() - 4 * 60 * 1000);
+export const MOCK_TRANSACTION_CONTRACT_CALL_MINUTES_AGO: MockTransaction = {
+  transaction: MOCK_TRANSACTION_CONTRACT_CALL.transaction,
+  tx: {
+    ...MOCK_TRANSACTION_CONTRACT_CALL.tx,
+    time: fourMinutesAgo.toString(),
+    id: '0x18617ccc580478214175c4daba11903df93a66a94aada773e80411ed06b6ade9',
+    status: TxStatus.failure,
+  },
+  receipts: MOCK_TRANSACTION_CONTRACT_CALL.receipts,
+};
+
+const twoHoursAgo = new Date(new Date().getTime() - 2 * 60 * 60 * 1000);
+export const MOCK_TRANSACTION_CONTRACT_CALL_HOURS_AGO: MockTransaction = {
+  transaction: MOCK_TRANSACTION_CONTRACT_CALL.transaction,
+  tx: {
+    ...MOCK_TRANSACTION_CONTRACT_CALL.tx,
+    operations: [
+      {
+        ...MOCK_TRANSACTION_CONTRACT_CALL.tx.operations[0],
+        name: OperationName.receive,
+      },
+    ],
+    time: twoHoursAgo.toString(),
+    id: '0x18617ccc580478214175c4daba11903df93a66a94aada773e80411ed06b6adea',
+  },
+  receipts: MOCK_TRANSACTION_CONTRACT_CALL.receipts,
+};
```

Comment:
You can create a function to abstract the creation of the tx and just pass the data you need to change will remove some of duplicated code here.

Reply:
hmm, thanks for this suggestion, I did that.